graphdatascience/graph_data_science.py

```diff
@@ -117,10 +118,16 @@ def __init__(
         self._query_runner.set_show_progress(show_progress)
         super().__init__(self._query_runner, namespace="gds", server_version=self._server_version)
 
+        self._wcc_endpoints = WccCypherEndpoints(self._query_runner)
```

Comment:
we could also only expose them on AuraGraphDataScience for now?

as for normal GDS Plugin the v2 endpoints are not available yet